tests/lib/tools/snaps-state

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+show_help() {
+    echo "usage: make-snap [snap-name]"
+    echo "usage: install-local [snap-name]"
```

Comment:
Please remove the subsequent repetitions of `usage: `

Reply:
done